src/gsrand.cpp

```diff
@@ -381,7 +381,7 @@ void filter_default_content(vector<string>& unfiltered, const char ** default_li
 							vector<string>& search_paths, string ext, int& total_count, int& exclude_count, string& last_print)
 {
 	uint64 last_print_time = 0;
-	if (contentMode != CONTENT_EVERYTHING || maxContentBytes && search_paths.size() || skipUppercase)
+	if ((contentMode != CONTENT_EVERYTHING || maxContentBytes) && (search_paths.size() || skipUppercase))

```

Comment:
Could you change it to this to preserve the original logic:
`(contentMode != CONTENT_EVERYTHING || (maxContentBytes && search_paths.size()) || skipUppercase)`

Everything else looks good.